pubsub/jetstream/jetstream.go

```diff
@@ -88,8 +88,29 @@ func (js *jetstreamPubSub) Features() []pubsub.Feature {
 }
 
 func (js *jetstreamPubSub) Publish(req *pubsub.PublishRequest) error {
-	js.l.Debugf("Publishing topic %v with data: %v", req.Topic, req.Data)
-	_, err := js.jsc.Publish(req.Topic, req.Data)
+	var opts []nats.PubOpt
+	var msgID string
+
+	event, err := pubsub.FromCloudEvent(req.Data, "", "", "", "")
```

Comment:
Nice use of this existing function. Glad it works for you here.

Reply:
Done